CRM/Event/Form/Registration/Confirm.php

```diff
@@ -45,6 +45,29 @@ class CRM_Event_Form_Registration_Confirm extends CRM_Event_Form_Registration {
    */
   protected $submittableMoneyFields = ['total_amount', 'net_amount', 'non_deductible_amount', 'fee_amount', 'tax_amount', 'amount'];
 
+  /**
+   * @var int|mixed
+   */
+  private $_amount;
+
+  /**
+   * Provide support for extensions that are used to being able to retrieve _lineItem
+   *
+   * Note extension should call getPriceSetID() and getLineItems() directly.
```

Comment:
@eileenmcnaughton This comment seems like a copy-paste from elsewhere.

Reply:
@braders snapped